src/util/std_expr.h

```diff
@@ -354,6 +354,11 @@ inline bool can_cast_expr<unary_exprt>(const exprt &base)
   return base.operands().size() == 1;
 }
 
+inline void validate_expr(const unary_exprt &value)
```

Comment:
:pick: commit message: unnecessariliy -> unnecessarily

Reply:
:pick:  some commit messages are too long

Reply:
Both done and automated spell checking enabled in my editor...